Tools/Cli-LoRa-Device-Provisioning/CommandLineOptions/AddGatewayOption.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tools.CLI.Options
+{
+    using System;
+    using CommandLine;
+
+    [Verb("add-gateway", HelpText = "Add a new gateway device to IoT Hub.")]
+    public class AddGatewayOption
+    {
+        internal const int DefaultSpiSpeed = 8;
+        internal const int DefaultSpiDev = 0;
+        internal const string DefaultAzureIotEdgeVersion = "1.4";
```

Comment:
We would need to add this occurence to the playbook to not forget to update it when we updat iotedge version

Reply:
Will be automated here #1928